### ISSUE TITLE
rpm-ostree: recommend packages from test fmf, causing installation to fail when package not found.

### ISSUE THREAD

**Issue by sbertramrh** [teemtee/tmt#1563]
```
        summary: Install recommended packages
        name: recommends
        package: policycoreutils-python, python2-lxml and jfsutils
    finish
        summary: 0 tasks completed

stdout (20/20 lines)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Checking out tree bcf8a8f...done
Enabled rpm-md repositories: extras-common baseos appstream crb beaker-client crb baseos-debug appstream-debug crb-debug autosd-debug epel baseos-source appstream-source baseos appstream automotive autosd
Importing rpm-md...done
rpm-md repo 'extras-common' (cached); generated: 2022-09-20T12:23:00Z solvables: 37
rpm-md repo 'baseos' (cached); generated: 2022-09-19T17:52:32Z solvables: 3017
rpm-md repo 'appstream' (cached); generated: 2022-09-19T17:54:21Z solvables: 10078
rpm-md repo 'crb' (cached); generated: 2022-09-19T17:55:16Z solvables: 1971
rpm-md repo 'beaker-client' (cached); generated: 2022-06-03T13:40:35Z solvables: 21
rpm-md repo 'crb' (cached); generated: 2022-09-19T17:55:16Z solvables: 1971
rpm-md repo 'baseos-debug' (cached); generated: 2022-09-19T17:28:07Z solvables: 869
rpm-md repo 'appstream-debug' (cached); generated: 2022-09-19T17:25:39Z solvables: 2648
rpm-md repo 'crb-debug' (cached); generated: 2022-09-19T17:29:11Z solvables: 312
rpm-md repo 'autosd-debug' (cached); generated: 2022-09-15T14:53:27Z solvables: 249
rpm-md repo 'epel' (cached); generated: 2022-09-22T00:30:56Z solvables: 11519
rpm-md repo 'baseos-source' (cached); generated: 2022-09-19T18:06:57Z solvables: 1217
rpm-md repo 'appstream-source' (cached); generated: 2022-09-19T18:08:50Z solvables: 3238
rpm-md repo 'baseos' (cached); generated: 2022-09-19T17:52:32Z solvables: 3017
rpm-md repo 'appstream' (cached); generated: 2022-09-19T17:54:21Z solvables: 10078
rpm-md repo 'automotive' (cached); generated: 2022-09-15T04:35:00Z solvables: 231
rpm-md repo 'autosd' (cached); generated: 2022-09-15T14:50:17Z solvables: 1411

stderr (1/1 lines)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
error: Packages not found: jfsutils, policycoreutils-python, python2-lxml
```

[sbertram@sbertram kernel-tests]$ tmt --version
tmt version: 1.17.0 (https://github.com/teemtee/tmt/commit/255c0918944589776cd10c2f9c418404fdf447ea)

**Comment by lukaszachy** [comment]
Copy&pasting @sopos' reproducer to verify that part as well:

> I face tmt failrue when trying to run a test which requires the `library(epel/epel)` which recommends `library(distribution/epel-internal)`.

**Comment by BeeGrech** [comment]
Quick debug, looks like self.skip is not being set for recommends in the rpm-ostree installer class
missing  is being set to list of packages, should be skip if true
https://github.com/teemtee/tmt/blob/main/tmt/steps/prepare/install.py#L295
.get('missing" vs .get('packages')

**Comment by BeeGrech** [comment]
https://github.com/teemtee/tmt/pull/1564

**Comment by psss** [comment]
Should be fixed by #1564.

**Comment by sbertramrh** [comment]
@psss confirming :

```
        how: install
        summary: Install recommended packages
        name: recommends
        package: rt-setup
        warn: No match for recommended package 'rt-setup'.
    
        summary: 3 preparations applied
    execute
        how: tmt
        progress: 
```
Thanks!
